java/src/Ice/src/main/java/com/zeroc/Ice/Blobject.java

```diff
@@ -27,15 +28,11 @@ public interface Blobject extends com.zeroc.Ice.Object {
   com.zeroc.Ice.Object.Ice_invokeResult ice_invoke(byte[] inEncaps, Current current)
       throws UserException;
 
-  /**
-   * @hidden
```

Comment:
The new dispatch is public and documented, so there is no reason to hide its Override in the generated code.